src/superphot_plus/format_data_ztf.py

```diff
@@ -107,7 +107,52 @@ def tally_each_class(labels):
     print()
 
 
-def oversample_using_posteriors(ztf_names, labels, chis, goal_per_class):
+def get_lightcurve_posterior_samples(ztf_name, fits_dir=None):
```

Comment:
See file_utils.get_posterior_samples introduced in PR #52.

Sorry for the duplication of effort!

Reply:
Ooh, nice catch! I erased the duplicated code to reuse the methods you introduced.